pkg/imgutil/filtering.go

```diff
@@ -145,18 +145,34 @@ func FilterByCreatedAt(ctx context.Context, client *containerd.Client, before []
 			maxTime = time.Now()
 		)
 
+		parseImageName := func(names []string) string {
+			var name string
+			if len(names) > 1 {
+				name = names[1]
+			} else {
+				name = names[0]
+			}
```

Comment:
can you explain why we use `names[1]` (not 0, not >1) if len(names) > 1?

Reply:
If you run command `nerdctl images -f since=nginx`. Then the first item is `docker.io/library/nginx:latest`, and the second item is `nginx`

https://github.com/containerd/nerdctl/blob/427f1cb6697af2a5c8d17550fb58d9f09aaa0a9e/pkg/imgutil/filtering.go#L96-L97

Reply:
how about lets just use `strings.Join(names, ",")`? given this is what we're actually doing.

Reply:
`FATA[0000] No such image: docker.io/library/aaaa:latest,aaaa:latest`. I don't know if this is better or not?